### ISSUE TITLE
Feedback on how to improve demo experience

### ISSUE THREAD

**Issue by tayloramurphy** [meltano/meltano-codespace-ready#6]
Douwe shared some feedback given from Sid:

[] "**copy and paste doesn't work**" - seems like it might not be as easy as we think to copy and past the config. worth testing. last week I tried to make the readme have those little copy to clipboard buttons but I don't know if that's possible in markdown or codespace. perhaps we need another solution here.
[] "**had to cd into the directory**" - he had to cd into the directory to get it to work. not sure if that's accurate but worth checking
[] "**first the WOW then the how**" - I _really_ like this feedback. @sbalnojan what do you think of updating the container so that the first thing a user runs is `meltano run` and then we can break down what's happening and / or have them do their own project?
[] "**paste this below pip_url can be more explicit**" - I think I wrote this - the language isn't _super_ clear on where to actually cut and paste.

**Comment by tayloramurphy** [comment]
@sbalnojan thinking on the "wow" factor a bit more - I wonder if there's some simple CLI visualization tool that we can use along with DuckDB to create a simple chart that outputs to the command line. 

I'm thinking we can have the user run 2 commands - a full `meltano run` that does the extract load and transformation. And then a python script (also via meltano run) that queries duckdb and outputs a chart based on that data to the command line. (maybe with a tool like this https://github.com/mkaz/termgraph ? that's a random repo I found). 

I'm worried that just having them run an end to end meltano run isn't a full enough wow, but that if we can show a cool image in the terminal and then walk them through the pipeline that would be a good how.

**Comment by DouweM** [comment]
> I just tried it and it seems to be working without an CD'ing.

@sbalnojan You're totally right, no need to cd into anything, but then the `meltano init` output is incorrect as it says "Next steps: cd new_project"

**Comment by sbalnojan** [comment]
@DouweM good point, maybe I can make the output I have in there anyways bigger and fatter :-D (Like don't cd,go straight to step 2, we already got your covered).

@tayloramurphy, I think that would again distract. the magic is in "meltano run tap map target". That's it.

I think it's fine to have a second step to make it visible. With a line like: "Wait we're already done? That's it??? Yes! The beauty of meltano is that it is soooo simple that you need a second step to make it visible".

**Comment by tayloramurphy** [comment]
@sbalnojan I trust your judgement on this! I think I'm just trying to avoid having the magic be in reading a bunch of log lines.
